src/pages/Analysis/index.js

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import {
+  Grid,
+  GridItem,
+  Card,
+  TextContent,
+  Text,
+  TextVariants,
+  Button,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import styles from './index.less';
+import AnalysisChart from '@/components/AnalysisChart';
+import { donutChart1, donutChart2, barGraph1, barGraph2 } from '../../../mock/analysis';
```

Comment:
Similar question as before, is this a mock component or the real version?

Reply:
The Analysis page is a real component which contains chart generated with mock data.

Reply:
Right, but why does the real code reference the `mock` module?